src/encoded/region_search.py

```diff
@@ -711,6 +725,19 @@ def format_results(request, hits, result=None):
                 item['highlight'] = {}
                 for key in hit['highlight']:
                     item['highlight'][key[9:]] = list(set(hit['highlight'][key]))
+
+            filtered_files = []
+            for file_ in item['files']:
+                filtered_file = {}
+                for field in REGION_SEARCH_FILE_FIELDS:
+                    if field in file_:
+                        filtered_file[field] = file_[field]
+                filtered_files.append(filtered_file)
+            item['files'] = filtered_files
+
+            for field in REGION_SEARCH_UNNECESSARY_FIELDS:
+                item.pop(field)
+
```

Comment:
Pulling specific fields from the `embedded` doc from Elasticsearch turns out to be slower than just getting everything and filtering out in the backend.